sklearn/svm/classes.py

```diff
@@ -1155,6 +1155,14 @@ class OneClassSVM(BaseLibSVM, OutlierMixin):
         The offset is the opposite of `intercept_` and is provided for
         consistency with other outlier detection algorithms.
 
+    Examples
+    --------
+    >>> from sklearn.datasets import make_circles
+    >>> from sklearn.svm import OneClassSVM
+    >>> X, y = make_circles(n_samples=6, factor=0.4, random_state=41)
```

Comment:
Might be clearer without shuffle? It's a bit confusing that we're getting y here, too.

Also simple would be to define a 1d dataset: [[-1], [0], [0.5], [1], [10]] or some such.

Reply:
thank you for the advice @jnothman, done that, but with a different array